randovania/games/samus_returns/layout/msr_configuration.py

```diff
@@ -17,6 +17,7 @@ class MSRArtifactConfig(BitPackDataclass, JsonDataclass):
 @dataclasses.dataclass(frozen=True)
 class MSRConfiguration(BaseConfiguration):
     energy_per_tank: int = dataclasses.field(metadata={"min": 1, "max": 1000, "precision": 1})
+    starting_energy: int = dataclasses.field(metadata={"min": 99, "max": 1000, "precision": 1})
```

Comment:
We can't start with less than 99 energy?

Reply:
Well, you can but I thought about be even more restrictive as the game UI is bad for it.
E.g. this is 130 energy...it won't show the purple bar until you have x99 energy.
![image](https://github.com/randovania/randovania/assets/117127188/28daa695-b258-44ca-89a2-aa6beca8bd58)